smartpower3/smartpower3.h

```diff
@@ -16,7 +18,10 @@ Button button[4] = {
 	Button(BUTTON_CH0), Button(BUTTON_CH1), Button(BUTTON_MENU), Button(BUTTON_DIAL)
 };
 
-Screen screen;
+ScreenManager screen_manager = ScreenManager();
+Settings settings = Settings();
+WiFiManager *wifi_manager;
```

Comment:
Settings and WIfiManager moved to global context to be accessible from almost anywhere.